alpha/veneer/basic/basic.go

```diff
@@ -0,0 +1,101 @@
+package basic
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-registry/alpha/action"
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+	"github.com/operator-framework/operator-registry/alpha/property"
+	"github.com/operator-framework/operator-registry/pkg/image"
+)
+
+var ErrNotAllowed = errors.New("not allowed")
+var ErrCannotIngestDirectory = errors.New("cannot ingest veneers by directory reference")
```

Comment:
Do we need to check for these specific errors anywhere? If not, I'd suggest either unexporting them or calling `errors.New()` directly in the return statement.